src/main/java/com/dev/cinema/controller/AuthenticationController.java

```diff
@@ -15,7 +16,8 @@ public AuthenticationController(AuthenticationService authenticationService) {
     }
 
     @PostMapping("/registration")
-    public void registerUser(@RequestBody UserRequestDto userRequestDto) {
-        authenticationService.register(userRequestDto.getEmail(), userRequestDto.getPassword());
+    public User registerUser(@RequestBody UserRequestDto userRequestDto) {
```

Comment:
```suggestion
    public void registerUser(@RequestBody UserRequestDto userRequestDto) {
```
void is ok) later we will return status code

Reply:
ой, это я проверяла как @RequestBody и @ResponceBody работают в одном методе и забыла вернуть все обратно) Исправила